corehq/motech/dhis2/tests/test_entities_helpers.py

```diff
@@ -220,14 +223,23 @@ def setUp(self):
     def test_no_relationships(self):
         requests = object()
         case_trigger_info = get_case_trigger_info_for_case(
-            self.child_case,
-            [{'case_property': 'external_id'}, {'case_property': 'dhis2_org_unit_id'}]
+            self.child_case, [{
+                'case_property': 'external_id'
+            }, {
+                'case_property': 'dhis2_org_unit_id'
+            }]
         )
         subcase_config = Dhis2CaseConfig.wrap({
-            'case_type': 'child',
-            'te_type_id': 'person12345',
-            'tei_id': {'case_property': 'external_id'},
-            'org_unit_id': {'case_property': 'dhis2_org_unit_id'},
+            'case_type':
+                'child',
```

Comment:
Urgh. Is this also yapf? I do like yapf, but not when it does things that a sane person would not do.